dmoj/cptbox/isolate.py

```diff
@@ -185,11 +184,9 @@ def __init__(self, read_fs, write_fs=None, writable=(1, 2)):
 
     def _compile_fs_jail(self, fs):
         if fs:
```

Comment:
Could probably replace this entire function with `return FilesystemPolicy(fs or [])`.